dotCMS/src/main/java/com/dotcms/publisher/business/PublisherAPIImpl.java

```diff
@@ -412,7 +412,7 @@ private void appendError ( List<String> errorsList, ErrorType errorType, User us
 		return res;
 	}
 
-	private static final String MULTI_TREE_QUERY = "select multi_tree.* from multi_tree join htmlpage_version_info on htmlpage_version_info.identifier = multi_tree.parent1 join container_version_info on container_version_info.identifier = multi_tree.parent2 join contentlet_version_info on contentlet_version_info.identifier = multi_tree.child where multi_tree.child = ? and htmlpage_version_info.deleted = ? and container_version_info.deleted = ? and contentlet_version_info.deleted = ?";
```

Comment:
Here, instead of deleting htmlpage_version_info and leave the query like that, it is needed to replace it with contentlet_version_info (for parent1). Use an alias called "page_version_info" to avoid errors in the sql